misc/python/materialize/mzcompose/services/ssh_bastion_host.py

```diff
@@ -28,10 +29,18 @@ def __init__(
             MZ_ROOT / "misc" / "images" / "sshd" / "setup.sh",
             loader.composition_path,
         )
+
+        # The current tagged release of this image doesn't support arm,
+        # so we override to a weekly tag for local users on darwin macs
+        if platform.machine() in ["aarch64", "arm64"]:
+            image = "panubo/sshd:weekly"
+        else:
+            image = "panubo/sshd:1.5.0"
```

Comment:
This is a hard-to-debug surprise waiting to happen. As a general rule, we should avoid this kind of conditional in any of our tests. A better option in general is to use Docker's SHA pinning (`panubo/sshd:weekly@sha256:...`) to use the _same_ pinned recent version on both platforms.

In this specific case, we can do even better. There _are_ new releases of panubo/sshd. They're just only on Quay/ECR, rather than on Docker Hub.

Reply:
nice, ill switch to that